src/registrar/models/domain.py

```diff
@@ -723,6 +723,7 @@ def _update_epp_contact(self, contact: PublicContact):
             fax=contact.fax,
         )  # type: ignore
 
+        updateContact.dislcose = self._disclose_fields(contact=contact)  # type: ignore
```

Comment:
```suggestion
        updateContact.diclose = self._disclose_fields(contact=contact)  # type: ignore
```

Reply:
OH I never pushed my last commit eek! thanks!

Reply:
pushed the change re-requesting review